node/blockAPI/metaBlock.go

```diff
@@ -15,8 +17,13 @@ type metaAPIBlockProcessor struct {
 }
 
 // NewMetaApiBlockProcessor will create a new instance of meta api block processor
-func NewMetaApiBlockProcessor(arg *APIBlockProcessorArg) *metaAPIBlockProcessor {
+func NewMetaApiBlockProcessor(arg *APIBlockProcessorArg) (*metaAPIBlockProcessor, error) {
 	hasDbLookupExtensions := arg.HistoryRepo.IsEnabled()
+	statusComputer, err := transaction.NewStatusComputer(arg.SelfShardID, arg.Uint64ByteSliceConverter, arg.Store)
```

Comment:
create _statusComputer_ outside of `metaAPIBlockProcessor` and put in `APIBlockProcessorArg` as a member of this structure. `metaAPIBlockProcessor` should receiver `statusComputer` as an argument

Reply:
Done!